src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -117,53 +131,57 @@ public Command rollerStop() {
         });
     }
 
-    public Command openArms() {
-        return new Command("Open Arms", this) {
-            @Override
-            protected void execute() {
-                pivotPointsMotor.set(.2);
-            }
+    public Command gamepieceCheck() {
+        // This command will pick up a ball
```

Comment:
Will it, though? Or will it just wait until a game piece is in position?

Reply:
Rohan sits on a seat of lies

Reply:
or andrew, just salty about fixing rohans space